docs/UserGuide.md

```diff
@@ -402,6 +402,24 @@ Exits the program.
 
 Format: `exit`
 
+### Go to previous commands: Press UpperArrow icon(PgUp)/ Press DownArrow icon(PgDn)
+Suppose the commands entered are: 
+Command 1: list
+Command 2: edit John Doe n/ John
```

Comment:
Would be better to remove the space after the prefix for consistency.
For instance change edit John Doe n/ John to edit John Doe n/John